packages/amazonq/.c8rc.json

```diff
@@ -2,6 +2,5 @@
     "report-dir": "../../coverage/amazonq",
     "reporter": ["lcov"],
     "all": true,
-    "include": ["src/**", "dist/src/**"],
```

Comment:
As far as I can tell/understand include isn't really necessary since we're using the "all" flag which considers all files in the cwd, --allowExternal which allows for src files to be outside of the amazonq directory (so it can see and report on the amazonq/codewhisperer test files), and exclude which specifically excludes certain files from the report